schemas/imaging/imaging_processing.json

```diff
@@ -0,0 +1,423 @@
+                },
+                "additional_params": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    },
+                    "title": "Additional parameters"
+                },
+                "host_file": {
+                    "type": "string",
+                    "title": "Host file"
+                },
+                "number_processes": {
+                    "type": "integer",
+                    "title": "Number processes"
+                }
+            },
+            "required": [
+                "image_depth",
+                "estimate_processes",
+                "additional_params",
+                "host_file",
+                "number_processes"
+            ]
+        },
+        "data_process":{
+            "type": "object",
+            "properties": {
+                "version": {
+                    "type": "string",
+                    "title": "Version",
+                    "description": "Version of the software used"
+                },
+                "start_date_time": {
+                    "type": "string",
+                    "format": "date-time",
+                    "title": "Start date time"
+                },
+                "end_date_time": {
+                    "type": "string",
+                    "format": "date-time",
+                    "title": "End date time"
+                },
+                "input_location": {
+                    "type": "string",
+                    "title": "Input location",
+                    "description": "Path to data inputs"
+                },
+                "output_location": {
+                    "type": "string",
+                    "title": "Output location",
+                    "description": "Path to data outputs"
+                },
+                "code_url": {
+                    "type": "string",
+                    "title": "Code URL",
+                    "description": "Path to code respository"
+                },
+                "notes": {
+                    "type": "string"
+                },
+                "name": {
+                    "anyOf": [
+                        {
+                            "title": "import",
+                            "type": "obejct",
+                            "properties": {
+                                "channel_excitation": {
+                                    "type": "integer",
+                                    "title": "Channel excitation (nm)",
+                                    "units": "nm"
+                                },
+                                "channel_emission": {
+                                    "type": "integer",
+                                    "title": "Channel emission (nm)",
+                                    "units": "nm"
+                                },
+                                "reference_1": {
```

Comment:
I assume this is what terastitcher wants, but I'm wondering if we can reuse the coordinate array format from the other schemas.

Reply:
it's too different from the coordinate array we have for ephys, but I do think I can make a subschema for this specifically...